src/theme.js

```diff
@@ -76,18 +76,12 @@ const flatten = (name, colors) => colors
     return {...a, ...color}
   }, {})
 
-const tints = [
-  0.2,
-  0.4,
-  0.6,
-  0.8
-]
-const blues = tints.map(tint(blue))
-const grays = tints.map(tint(gray))
-const greens = tints.map(tint(green))
-const reds = tints.map(tint(red))
-const oranges = tints.map(tint(orange))
-const purples = tints.map(tint(purple))
+const blues = [brighten(blue), blue, darken(blue)]
```

Comment:
It sounded like @oliverdumoulin wasn't using a color function for his updated color scale anymore, so I think we'll want to remove the chroma-js dependency and just hard code the values in

Reply:
@jxnblk We are depending on the tint function (depending on chroma) in the legacy theme. Should we hard code those values as well?

Reply:
Yeah, I think that's fine – probably worth getting rid of that dependency if we can